doc/Ischia Developer Docs/Feature Overview.md

```diff
@@ -0,0 +1,96 @@
+# Feature Overview
+
+### Campaigns Type
+
+| Type | Description | State |
+| --- | --- | --- |
+| Daily task | Activate or log in once a day  — offchain |  |
+| Online time | On line at a preset time  |  |
+| Number of Transactions | Whether the number of trades on a contract has reached a preset number |  |
+| Daily - Interaction or not | Whether the user has interacted with a contract  |  |
+| Follow | Follow x or join discord or github |  |
+|  |  |  |
+
+
+### 3. Daily Activity
+
+**Description**: Activate or log in once a day
+
+**Function Description**:
+
+- Users are required to log in to the platform daily to maintain their activation status.
+- This encourages regular engagement and activity on the platform.
+
+**Implementation Steps**:
+
+1. Users log in to the platform daily.
+2. The platform records the login and updates the user's activation status.
+3. The platform may provide rewards or incentives for maintaining a daily login streak.
+
+**Application Scenarios**:
+
+- Encourages daily platform engagement and consistent user activity.
+- Helps maintain an active user base and can be tied to reward programs.
```

Comment:
Assists in retaining an engaged user community and can be linked to incentive schemes.